Rakefile

```diff
@@ -1,6 +1,11 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
+require 'graphql/rake_task'
 
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
```

Comment:
what does this do?

Reply:
This is the rake task that created the schema.graphql file and the schema.json file